src/CLI/IOptions.cs

```diff
@@ -12,5 +12,6 @@ internal interface IOptions
         VerbosityLevel VerbosityLevel { get; }
         int DelayInSeconds { get; }
         string CustomUia { get; }
+        bool ScanMultipleWindows { get; }
```

Comment:
Is it possible to make this name more descriptive, especially since it's publicly exposed? Maybe EnableMultipleScanRoots or something along those lines?

Reply:
Went with EnableMultipleScanRoots, I think it's a very fitting name.